Website/src/rendering/src/stories/Page Content/Section.stories.tsx

```diff
@@ -10,23 +10,26 @@ export default {
   component: Section,
```

Comment:
You have added additional Section CSS classes for light-pattern and dark-pattern. Please add additional Storybook variants for them here.